test_unstructured_ingest/test-ingest-box-dest.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+#TODO currently box api/sdk does not work to create folders and check for content similar to other fsspec ingest tests
```

Comment:
for the purposes of testing: can we just attempt to download the content in the test and confirm that we can fetch it?

Reply:
The problem here is I can't even create a temporary path with confidence to use for testing in the same way I create it everywhere else. We could technically push to a new path but then have no way to clean it up at the end, bloating how much we store in the box account.

Reply:
Well across the board, it's aws cli for s3, `az` for azure and `gcloud` for google gloud storage.

Reply:
For dropbox it had to be done through their api using `curl`